external/avro/benchmarks/AvroReadBenchmark-jdk11-results.txt

```diff
@@ -2,121 +2,121 @@
 SQL Single Numeric Column Scan
 ================================================================================================
 
-OpenJDK 64-Bit Server VM 11.0.4+11-LTS on Linux 3.10.0-862.3.2.el7.x86_64
-Intel(R) Xeon(R) CPU E5-2670 v2 @ 2.50GHz
+Java HotSpot(TM) 64-Bit Server VM 11.0.5+10-LTS on Linux 4.15.0-1051-aws
```

Comment:
Ur, is this not `OpenJDK`? For JDK8 result, this PR is using `OpenJDK 64-Bit Server VM 1.8.0_232-8u232-b09-0ubuntu1~18.04.1-b09`.

Reply:
I took JDK 8 from Ubuntu 18.04 but JDK 11 from Oracle website: https://www.oracle.com/technetwork/java/javase/downloads/jdk11-downloads-5066655.html